_posts/2018-04-03-bug-bbq-how-to-be-involved.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+permalink: 2018-04-03-bug-bbq-involved
+subheadline: "What can I do during the Bug BBQ?"
+title: "How to be involved in the Bug BBQ?"
+teaser: "A short guide on how to contribute and be involved during The Carpentries Bug BBQ."
+header:
+    image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+    - blog
+comments: true
+show_meta: true
+authors: ["François Michonneau"]
+---
+
+
+## I want to contribute to the lessons during the Bug BBQ!
```

Comment:
A reminder at the top of the blog post about when the Bug BBQ will be with a link to the time and date would be good. Website link can go here too.